### PR TITLE
Switch remaining Fprintf(os.stderr) to stderr()

### PR DESCRIPTION
**Pull request by jbergstroem**
Because nitpick

### CODE REVIEW

File: actool/discover.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func init() {
 
 func runDiscover(args []string) (exit int) {
 	if len(args) < 1 {
-		fmt.Fprintf(os.Stderr, "discover: at least one name required")
+		fmt.Fprintf(os.Stderr, "discover: at least one name required\n")
```
Comment (jonboulle) [inline actool/discover.go]: Thanks for the fix. Would you mind actually just changing this to use the [`stderr` helper function](https://github.com/appc/spec/blob/master/actool/actool.go#L99) for consistency with elsehwere?

Comment (jbergstroem) [inline actool/discover.go]: Sure. Will update shortly.

**Comment by jbergstroem** [comment]
I found a few additional places where the same pattern were used and replaced them as well. I avoided some corner cases where newline wasn't expected. Hope that's fine.

File: ace/validator.go [hunk 2]
```diff
@@ -106,7 +106,7 @@ func main() {
 	case "poststop":
 		res = validatePoststop()
 	default:
-		fmt.Fprintf(os.Stderr, "unrecognized mode: %s\b", mode)
+		stderr("unrecognized mode: %s\b", mode)
```
Comment (jonboulle) [inline ace/validator.go]: Doh! The `\b` is a bug, could you remove it please?

**Comment by jbergstroem** [comment]
Fixed.

**Comment by philips** [comment]
lgtm, merging.

**Comment by jonboulle** [comment]
This jumped the gun a bit :/
```
gopath/src/github.com/appc/spec/ace/validator.go:94: undefined: stderr
```

**Comment by jbergstroem** [comment]
@jonboulle strange, I never got that error. Just tried building on both os x/linux with go 1.4 (ace not built on OS X).

**Comment by jonboulle** [comment]
@jbergstroem I subsequently fixed it in https://github.com/appc/spec/commit/a4cff3a3f49eeeecc5cd05ad80ede739b9590392 (and enabled travis for the repo, so we should catch these in future!)

**Comment by jbergstroem** [comment]
@jonboulle Got it. Thanks, appreciated. Fwiw, the issue was me not testing on linux (only os x) before putting my new round of patches up for review. Apologies.

**Comment by jonboulle** [comment]
@jbergstroem No worries, it's clearly a bug in our process that we didn't have travis enabled, so thanks for catching that ;-)

**Pull request merged**